src/pytest_mock_resources/fixture/database/generic.py

```diff
@@ -41,6 +41,13 @@ def as_mongo_kwargs(self):
             "authSource": self.database,
         }
 
+    def as_redis_kwargs(self):
```

Comment:
Does this produce a redis-compatible dsn string? If so, I suppose this is fine. Otherwise, I'd be a bit leary about bundling this here. We might want one which is specific to redis

Reply:
I believe so, this test tests the same -> https://github.com/schireson/pytest-mock-resources/pull/77/files#diff-3decb146c96bf9692bc318749da9e616R34

Reply:
are there implicit user/pass values that we could/should be sending then?

Reply:
We could, the `config` dict doesn't include a username and a password, since it not required to connect to Redis.
But we most definitely add them.

Reply:
I don't think so, they are set to `None` by default -> https://redis-py.readthedocs.io/en/latest/index.html#redis.Connection

Reply:
So long as returning them as `None` here (e.g. passing in `None` to the constructor, then just returning them here) doesn't break the `Redis(**...as_redis_kwargs()` then i feel like we should be returning them from this method